Capstone/Assets/Game/Scripts/Enemy_SideToSide.cs

```diff
@@ -15,26 +15,45 @@ public class Enemy_SideToSide : Character
         private bool damaged = false;
         private float timer = 0f;
         private float attackTimer = 0f;
-        private float moveTimer = 0f;
-        private int flip = 1;
+        private int index;
```

Comment:
Can this be changed to something more descriptive? Even like `positionIndex` or `movePointIndex` would work